spec/features/global_search_spec.rb

```diff
@@ -34,4 +34,34 @@
     end
   end
 
+  # see dummy app's authorization concern for auth mapping used here
+  scenario "search results get authorized", js: true do
```

Comment:
ok moved all this to work on all_models, i didn't realize it was already working on pages because the static items in the global search are built from the already-authorized nav structure. win-win